src/main/java/com/mitchej123/hodgepodge/asm/InventoryEffectRendererTransformer.java

```diff
@@ -0,0 +1,35 @@
+package com.mitchej123.hodgepodge.asm;
+
+import com.mitchej123.hodgepodge.asm.util.AbstractClassTransformer;
+import com.mitchej123.hodgepodge.asm.util.AbstractMethodTransformer;
+import org.objectweb.asm.Opcodes;
+import org.objectweb.asm.tree.AbstractInsnNode;
+import org.objectweb.asm.tree.InsnList;
+
+public class InventoryEffectRendererTransformer extends AbstractClassTransformer {
+    public InventoryEffectRendererTransformer()
+    {
+        methodTransformers.put(References.cEffectRederer.getMethod("drawScreen"), new AbstractMethodTransformer() {
+            @Override
+            public void transform() {
+                // Cut out super.drawScreen() sequence
+                AbstractInsnNode start = findNext(currentMethod.instructions.getFirst(), matchOpcode(Opcodes.ALOAD));
+                AbstractInsnNode[] instToMove = new AbstractInsnNode[5];
```

Comment:
This is really minor, but I think you could make this variable an InsnList and be able to delete lines 26-28 that way?

Reply:
It is not minor, maybe it deserves a comment: InstList takes ownership of the instruction wrapper, and it would break currentMethod.instructions list

Reply:
Ah, gotcha. Sorry, I'm unfamiliar with these libraries